pastes/models.py

```diff
@@ -1,7 +1,9 @@
+from django.contrib.postgres.fields import JSONField
 from django.db import models
 
 class Pastes(models.Model):
-    id = models.CharField(max_length=255, primary_key=True)
+    id = models.AutoField(primary_key=True, serialize=True)
```

Comment:
If it weren't for the `serialize=True` I would say this declaration is entirely redundant, as Django will add an AutoField called 'id' if you do not explicitly declare a primary key.

I am curious of your intent with the `serialize=True`, as it doesn't appear to be a documented argument.